### PR TITLE
Ticket 8 post comment

### PR DESCRIPTION
**Pull request by veramferreira**

### CODE REVIEW

File: src/components/PostComment.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import { useState } from "react";
+import { fetchCommentsByArticleId, postCommentByArticleId } from "../utils";
+import BeatLoader from "react-spinners/BeatLoader";
+
+export default function PostComment({ article_id, setComments }) {
+  const [newComment, setNewComment] = useState("");
+  const [isLoading, setIsLoading] = useState(false);
+  let [color, setColor] = useState("#ffffff");
```
Comment (JVal-NC) [inline src/components/PostComment.jsx:8]: Any reason this is let rather than const?

File: src/components/PostComment.jsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  function handleSubmit(event) {
+    event.preventDefault();
+    setIsLoading(true);
+
+    postCommentByArticleId(article_id, newComment)
+      .then((newCommentFromApi) => {
+        setComments((currComments) => {
+          return [newCommentFromApi, ...currComments];
+        });
+        setPostMessage("Boom! Comment successfully added! 🎉");
```
Comment (JVal-NC) [inline src/components/PostComment.jsx:20]: Nice feedback message :)

**Review by JVal-NC — COMMENTED** [review]
Vera great work! Go ahead and merge.

File: src/components/PostComment.jsx (hunk 1 above)
Comment (veramferreira) [reply src/components/PostComment.jsx:8]: not really. 😅 sorry. This is from when I copied the code from the docs.

**Pull request merged**